docs/dev/README.md

```diff
@@ -36,4 +38,8 @@ They may require some small tweaks for your environment (as it will be different
 4. deploy the application  `kubectl apply -f jenkins-kube-creds.yaml`
 5. deploy service so that Jenkins is exposed (optional and one time only)  `kubectl apply -f service.yaml`
 
-Note: [this page](https://cloud.google.com/container-registry/docs/pushing-and-pulling) is useful for setting up auth to push to GKE
\ No newline at end of file
+Note: [this page](https://cloud.google.com/container-registry/docs/pushing-and-pulling) is useful for setting up auth to push to GKE

```

Comment:
I would also add:
"In short: `gcloud docker -- push <image>`"

Reply:
added